docs/docsite/rst/community/maintainers_workflow.rst

```diff
@@ -1,20 +1,5 @@
 .. _maintainers_workflow:
 
-.. _Releasing:
-
-Releasing collections
-======================
-
-Collection maintainers release all supported stable versions of the collections regularly,
-provided that there have been enough changes merged to release.
-
-Generally, releasing in the collections consists of:
-
-1. Planning and announcement.
-2. Generating a changelog.
-3. Creating a release git tag and pushing it.
-4. Automatic publishing the release tarball on `Ansible Galaxy <https://galaxy.ansible.com/>`_ by `Zuul <https://dashboard.zuul.ansible.com/t/ansible/builds?pipeline=release>`_.
-5. Final announcement.
```

Comment:
Should we maybe keep the `Releasing collections` paragraph in `docs/docsite/rst/community/maintainers_workflow.rst` because it's one of the main maintainer tasks, i.e. a part of the workflow, so maybe it's worth mentioning it here?
Instead of the removed content, we could put something (meaning): "Collection maintainers release their collections. See ... for details" or "To learn more, see ...".
What do you think?

Reply:
yeah that's a good idea, thanks!